packages/core/src/components/portal/portal.tsx

```diff
@@ -87,3 +90,7 @@ export class Portal extends React.Component<IPortalProps, {}> {
         this.targetElement.remove();
     }
 }
+
+function maybeAddClasses(element: Element, ...classes: Array<string | undefined>) {
```

Comment:
nit: I'd be comfortable just calling this `addClasses` with the understanding that undefined classes won't be added, but this is fine too.